src/style.css

```diff
@@ -0,0 +1,95 @@
+@font-face {
+  src: url(fonts/Avenir-Book.ttf);
+  font-family: "Avenir-Book";
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-family: "Avenir-Heavy";
```

Comment:
```suggestion
  font-family: "Avenir";
```

Reply:
You can use same font-family for both @font-faces but different font-weight. Then changing only the font-weight on your elements will do the job.